public/javascripts/SVLabel/src/SVLabel/navigation/MapService.js

```diff
@@ -587,6 +587,15 @@ function MapService (canvas, neighborhoodModel, uiMap, params) {
         return uiMap.pano.find('svg').parent();
     }
 
+    /**
+     * Get link layer of bottom links.
+     * @returns {*}
+     */
+    function getBottomLinkLayer () {
+        var links = $('.gm-style-cc').slice(0, 2);
```

Comment:
It's just a style/preference thing, but I prefer just a one line `return $('.gm-style-cc').slice(0, 2);`. I just don't see the need to save it in a variable for a hot second.